README.md

```diff
@@ -17,7 +17,7 @@
 
 ## Application Page url
 
-https://phelian23.github.io/to-do-list/
+Here is a link to the [live](https://phelian23.github.io/to-do-list/) demo
 
 
 ## Getting Started
```

Comment:
Note for future projects:
**You must ensure that your pull request includes all commits related to the change you made. We are using the `Files Changed` feature of GitHub to see which files are updated between pull requests.**